zulip_bots/zulip_bots/bots/flock/fixtures/test_no_recipient_found.json

```diff
@@ -6,9 +6,12 @@
             "token": "12345"
         }
     },
-    "response": {
-        "error": "No user found. Make sure you typed it correctly."
-    },
+    "response": [
+        {
+            "id": 1,
```

Comment:
There's a typo in the commit message: 

`flock: Refactor test_no_recipient_found test to incrase coverage.` -> incrase to increase.

Reply:
@aero31aero Fixed